### DISCUSSION TITLE
An unexpected error happened when running this build

### DISCUSSION THREAD

**Discussion by txfs19260817** [vercel/community#1443] [Help] [answered]
### Summary

After building and before assigning domains, an error without details occurs: 

```
10:45:51.508 |Traced Next.js server files in: 15.249s
10:45:51.519 | Created all serverless functions in: 6.009s
10:45:51.920 | Collected static files (public/, static/, .next/static): 26.65ms
10:46:25.947 | Build Completed in /vercel/output [9m]
10:53:47.406 | Generated build outputs:
10:53:47.406 | - Static files: 262
10:53:47.406 | - Prerenders: 36404
10:53:47.407 | - Serverless Functions: 16
10:53:47.407 | - Edge Functions: 0
10:53:47.407 | Serverless regions: San Francisco, USA
10:53:47.407 | Deployed outputs in 382s
10:53:59.052 | An unexpected error happened when running this build. We have been notified of the problem. If you have any questions, please contact Vercel Support https://vercel.com/help.
```

Retried many times. Reverted to an old success build still not working.

### Additional information

My repo: https://github.com/txfs19260817/falinks-teambuilder

**Comment by amyegan** [comment, marked as answer]
Thanks for sharing the repo, @txfs19260817. It looks like your site is using [ISR](https://nextjs.org/docs/basic-features/data-fetching/incremental-static-regeneration) and pre-rendering a lot of pages at build time. Try to reduce the number of `paths` you pre-render in `getStaticPaths` and let them [generate on-demand](https://nextjs.org/docs/basic-features/data-fetching/get-static-paths#generating-paths-on-demand) instead
